.github/ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,13 @@
 <!--
 Thank you for reporting an issue.
 
+*IMPORTANT* - please search our issue tracker *first* to ensure the
+issue hasn't been reported before. Open a new issue *only* if you haven't
+found anything similar to your issue - https://github.com/zfsonlinux/zfs/issues 
+
 This issue tracker is for bugs and issues only.
-If you need general support please use 
-our mailing list. https://github.com/zfsonlinux/zfs/wiki/Mailing-Lists
+If you need general support please use our mailing list. 
```

Comment:
also, user should be directed to docs+wiki first, then search the mailing lists before being encouraged to write to list